<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mb_exchangeClass</class>
 <widget class="QMainWindow" name="mb_exchangeClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1268</width>
    <height>887</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>mb_exchange</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout" stretch="5,1">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="1,5">
        <item>
         <widget class="QTreeWidget" name="treeWidget">
          <column>
           <property name="text">
            <string>设备</string>
           </property>
          </column>
         </widget>
        </item>
        <item>
         <widget class="QTabWidget" name="tabWidget">
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="tab">
           <attribute name="title">
            <string>遥信</string>
           </attribute>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QTableWidget" name="tableWidget">
              <attribute name="horizontalHeaderStretchLastSection">
               <bool>true</bool>
              </attribute>
              <column>
               <property name="text">
                <string>遥信ID</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>厂站ID</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>设备ID</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>值</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>描述</string>
               </property>
              </column>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tab_2">
           <attribute name="title">
            <string>遥测</string>
           </attribute>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QTableWidget" name="tableWidget_3">
              <attribute name="horizontalHeaderStretchLastSection">
               <bool>true</bool>
              </attribute>
              <column>
               <property name="text">
                <string>遥测ID</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>厂站ID</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>设备ID</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>值</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>描述</string>
               </property>
              </column>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tab_3">
           <attribute name="title">
            <string>遥脉</string>
           </attribute>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QTableWidget" name="tableWidget_4">
              <attribute name="horizontalHeaderStretchLastSection">
               <bool>true</bool>
              </attribute>
              <column>
               <property name="text">
                <string>电度ID</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>厂站ID</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>设备ID</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>值</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>描述</string>
               </property>
              </column>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTableWidget" name="tableWidget_2">
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>true</bool>
        </attribute>
        <column>
         <property name="text">
          <string>日期</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>时间</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>设备ID</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>设备名</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>描述</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1268</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="mb_exchange.qrc"/>
 </resources>
 <connections/>
</ui>
